spk/gogs/src/dsm-control.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/sh
+#
+# with WIZARD_FILES select log file or not
+#
+
+# Package
+PACKAGE="gogs"
+DNAME="Gogs"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+DIR_GOGS="${INSTALL_DIR}/gogs"
+GOGS="${DIR_GOGS}/gogs"
+PID_FILE="/var/run/gogs.pid"
+LOG_FILE="/var/log/gogs.log"
+
+FILE_CREATE_LOG="${DIR_GOGS}/wizard_create_log"
+
+export HOME=${DIR_GOGS}
+#export PATH=$PATH:~/opt/bin  # to Git. Not necessary with Git Server (Synology)
+export USER=root
```

Comment:
running as root is never a good idea...
take a look at some other tools... create a user on install and use that one...